pkg/clusterconfig/clusterconfig.go

```diff
@@ -25,6 +30,8 @@ const (
 	installerConfigNamespace = "kube-system"
 	installerConfigName      = "cluster-config-v1"
 	installerAWSCredsName    = "aws-creds"
+	imageRegistryNamespace   = "openshift-image-registry"
```

Comment:
we don't have this somewhere else already?

Reply:
updated in next push